course_info/static/course_info/js/directives/HuDatePicker.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+    var app = angular.module('CourseInfo');
+    app.directive('huDatePicker', huDatePicker);
+
+    function huDatePicker() {
+        return {
+            restrict: 'E',
+            scope: {
+                editable: '<',  // if not provided, defaults to null/false
+                field: '@',
+                formValue: '=',
+                isLoading: '&',
+                label: '@',
+                modelValue: '=',
+                term: '=',
+                termConcludeDate: '=',
+                isSelectedDateInPast: '='
+            },
+            templateUrl: 'partials/hu-date-picker.html',
+            link: function (scope, element) {
+                $('#dp-alert').hide();
```

Comment:
Could `#dp-alert` become `.dp-alert`, and these linked hide/show actions could point to `$(element).find('.dp-alert')`, similar to what's done for `inputElement`, in order to make this 'reusable' in the sense of not having this directive-scoped code potentially refer to elements outside of its DOM subtree? Would that work?